modules/sso.php

```diff
@@ -148,7 +148,7 @@ public function login_body_class( $classes ) {
 		// Always add the jetpack-sso class so that we can add SSO specific styling even when the SSO form isn't being displayed.
 		$classes[] = 'jetpack-sso';
 
-		if ( ! Jetpack::is_staging_site() ) {
+		if ( ! ( new Status() )->is_staging_site() ) {
```

Comment:
I don't understand why we're using two invocation patterns in this PR: `( new Status() )->some_function()` vs `$status->some_function()`.

Am I missing something about where/how they should be used?

Reply:
When there's only one usage of the package class in a function, I've stuck with `( new Status()->some_function()`. When the same class is used multiple times, I've set it with a variable.

e.g. _inc/lib/class.core-rest-api-endpoints.php L981 and L984 in the same function call two separate status checks. Where modules/sso.php L151 is the only check within that function.

I don't know that it matters really since Status has no constructor, but in general, I would think it is more performant not to construct multiple class instances within the same function.